### PR TITLE
Json server

### LINKED ISSUE #5: JSON Server
# As a devoloper,
I want to create a mock database, so that I have the necessary data available without having a real backend.

# Description
JSON Server with fake data.

# Done
- [ ] Add JSON server
- [ ] Fill the server with data
- [ ] Connect the JSON server to the application

### PR DESCRIPTION
**Pull request by mschmauder-hub**
Fix #5 

Add JSON server and adjust a bit of Code, so it works on Android.

### CODE REVIEW

File: SeriesMaster-Frontend/App/Screens/DetailsScreen.js [hunk 1]
```diff
@@ -29,25 +29,33 @@ const TagsContainer = styled.View`
...
+const DetailsScreen = ({ route }) => {
   const [completed, setCompleted] = useState(false);
   const [onWatchList, setOnWatchList] = useState(false);
+  const [tvShow, setTvShow] = useState("");
+
+  useEffect(() => {
+    shows();
+  }, []);
+
+  async function shows() {
```
Comment (marwinburesch) [inline SeriesMaster-Frontend/App/Screens/DetailsScreen.js]: I think this function could have a more descriptive name :)

File: SeriesMaster-Frontend/App/Screens/DetailsScreen.js [hunk 2]
```diff
@@ -29,25 +29,33 @@ const TagsContainer = styled.View`
...
+    const show = await getShows(route.params.id);
```
Comment (marwinburesch) [inline SeriesMaster-Frontend/App/Screens/DetailsScreen.js]: Isn't this a singular show?

**Review by marwinburesch — APPROVED** [review]
Good PR, looks straight forward and logical. 🐜

File: SeriesMaster-Frontend/App/Screens/DetailsScreen.js (hunk 1 above)
Comment (mschmauder-hub) [reply SeriesMaster-Frontend/App/Screens/DetailsScreen.js]: Yes, definitely not the best name.

File: SeriesMaster-Frontend/App/Screens/DetailsScreen.js (hunk 2 above)
Comment (mschmauder-hub) [reply SeriesMaster-Frontend/App/Screens/DetailsScreen.js]: Yes, I will fix that. Thanks!

**Pull request merged**